lib/money_column/active_record_hooks.rb

```diff
@@ -26,7 +26,7 @@ def init_internals
     end
 
     module ClassMethods
-      def money_column(*columns, currency_column: nil, currency: nil, currency_read_only: false)
+      def money_column(*columns, currency_column: nil, currency: nil, currency_read_only: false, allow_nil: true)
```

Comment:
Naming wise, what do you think about `coerce_null`? It would also require changing the code in this PR but I think it's a bit more clear on what will happen.